packages/react-shared-contexts/src/TooltipContext/TooltipContext.ts

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { useFluent } from '../ProviderContext/ProviderContext';
+import { TooltipManager, TooltipManagerCreateFunction } from './TooltipManager.types';
+
+/**
+ * The context provided by TooltipProvider
+ *
+ * {@docCategory TooltipProvider}
+ */
+export type TooltipContext = {
```

Comment:
```suggestion
export type TooltipContextValue = {
```

Can you please rename this type as currently it collides with a variable name?